src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -33,6 +33,11 @@ object ExportVCF {
             sb += '.'
           else
             sb.append(x.formatted("%.5e"))
+        case TLong =>
+          val x = a.asInstanceOf[Long]
+          if (x > Int.MaxValue || x < Int.MinValue)
+            fatal(s"Cannot convert Long to Int if value is greater than Int.MaxValue or less than Int.MinValue. Found $x.")
```

Comment:
"Cannot convert Long to Int if value is greater than Int.MaxValue (2^31 - 1) or less than Int.MinValue (-2^31). Found $x."